Apps/Examples/Examples/All Examples/FrameViewAnnotationsExample.swift

```diff
@@ -0,0 +1,73 @@
+import UIKit
+import MapboxMaps
+
+final class FrameViewAnnotationsExample: UIViewController, ExampleProtocol {
+
+    private var mapView: MapView!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
```

Comment:
Nit: Add white color for the background so the navigation bar wouldn't look dirty.
```suggestion

        view.backgroundColor = .white

```